chatbot/test/fix-arg-qr-translation-test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const assert = require('assert');
+const modify = require('../insert/fix_arg_qr_translation.js');
+
+function readInputFile(filePath) {
+    return JSON.parse(fs.readFileSync(filePath).toString());
+}
+
+describe('Test of "Fixer Arg and QR after translation" function', function() {
+    it('should fix the arguments of various nodes to ensure the logic of the english is consistent with the translation, runs the function on a sample input and check it matches the ideal output file', function() {
```

Comment:
I like long-ish descriptions of tests, but these should still be kept succinct. "runs the function on a sample input and check it matches the ideal output file" is unnecessary and should be removed.

Reply:
Changed here and elsewhere